lib/dialogue-agent/notifications/email.ts

```diff
@@ -67,10 +92,41 @@ export default class EmailNotificationBackend {
             to = profile.email;
         }
 
+        let unsubscribeURL = interpolate(this._unsubURL, {
+            email: new Buffer(to).toString('base64')
+        }, {
+            locale: this._platform.locale,
+            timezone: this._platform.timezone
+        })!;
+
+        let text = data.formatted.map((x) => x.toLocaleString(this._platform.locale)).join('\n');
+        text += `
+----
+You are receiving this email because you subscribed to notifications
+from Genie. To stop, open the following link:
+${unsubscribeURL}
+`;
```

Comment:
where does it actually go to? where is the backend to unsubscribe?

Reply:
In versions of almond-cloud that have the email configuration enabled (aka genie.stanford.edu), there is an endpoint to unsubscribe: https://github.com/stanford-oval/almond-cloud/blob/wip/genie-for-covid/routes/my_api.js#L129